src/sentry/sentry_metrics/indexer/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import connections, models, router
+
+from sentry.db.models import BaseModel
+from sentry.db.models.fields.bounded import BoundedBigIntegerField
+
+
+class MetricsKeyIndexer(BaseModel):
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    key = models.CharField(max_length=200, primary_key=True)
+    value = BoundedBigIntegerField()
+
+    class Meta:
+        db_table = "sentry_metricskeyindexer"
+        app_label = "sentry"
+        indexes = [
+            models.Index(fields=["organization_id", "key"]),
+            models.Index(fields=["organization_id", "value"]),
+        ]
+
+    @classmethod
+    def get_next_values(cls, num: int):
```

Comment:
This is not used right now but was my thought process for later, to test out get n number of next values. This does load all the integers which may not be ideal, so the other option would be to have the increment be larger and we just get the range from the last val to the next val e.g currval is 100 and next is 200, when we have 101 -200 or whatever